.github/workflows/scripts/docs/update-versions-json.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
We might run shellcheck separately on this file in this workflow as it is not possible to define dependencies between workflow files. Therefore (like it is now) the deployment went through even though the lint of this file has multiple errors.

I will look at it when looking for the markdown lint

Reply:
This script never got run btw. The workflow in the PR won't trigger it as it's only run when a commit has been tagged, which doesn't work with pull requests IIRC. Perhaps if running on your fork it would (fork the branch, make a commit, tag it, and it should trigger a versions build instead of edge build).

Reply:
But if there would have been a tag the script would have run even though the lint of the file throws an error. I stumpled over that behavior more often than i would like to. I am really missing cross workflow dependencies/requirements.

Reply:
Github CI allows for [workflow being triggered based on the success of another workflow I think](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run).

Reply:
Yeah I know but the types are only “requested” and “completed”. If you want to test if previous run was successful you need to use `if: ${{ github.event.workflow_run.conclusion == 'success' }}` which will lead to a lot of these "skipped" workflows that are seen on PRs for example.

Reply:
The alternative is having the lint workflow trigger the deploy workflow instead, which would avoid the skipped concern. But I don't think that's appropriate for Push event. With PR event linting and build verification can run separately too. Thus I think the concern is only due to the development here vs future maintenance which should always being going through the PR process?